Gemfile

```diff
@@ -3,15 +3,15 @@ source 'https://rubygems.org'
 gem 'berkshelf'
 
 group :style do
+  gem 'foodcritic', '~> 6.0.1'
```

Comment:
just a nit, but why the reorder?

Reply:
Rubcop wants it in alphabetical ordering.

Reply:
ah; makes sense, although a comment in the commit would have saved me some typing :p.

Reply:
That's fair. WIll add more info to the commit message for history.

The EOF/H change was to make rubocop happy again. It filters out EO* delimiters and warns about using a more meaningful one.